x-pack/plugins/cloud_security_posture/common/utils/helpers.ts

```diff
@@ -172,3 +173,35 @@ export const cleanupCredentials = (packagePolicy: NewPackagePolicy | UpdatePacka
   // nothing to do, return unmutated policy
   return packagePolicy;
 };
+
+export const getBenchmarkCisName = (benchmarkId: BenchmarksCisId) => {
+  switch (benchmarkId) {
+    case 'cis_k8s':
+      return 'CIS Kubernetes';
+    case 'cis_azure':
+      return 'CIS Azure';
+    case 'cis_aws':
+      return 'CIS AWS';
+    case 'cis_eks':
+      return 'CIS EKS';
+    case 'cis_gcp':
+      return 'CIS GCP';
+  }
+  return null;
+};
```

Comment:
in the last sync it was discussed we should use the `benchmark.name` field value and not hard code it here

Reply:
yesterday in the sync, I asked this question and Tinsae says that we are sticking with the design on Figma (which is showing the value listed above) 
He mentioned that last week we should be using the name because he thought that the name also includes the Benchmark version (check out benchmark.name for k8s)

Reply:
hmmm I see.. ok lets move with it as it is then